packages/protocol/scripts/foundry/start_op_anvil.sh

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+# This script is meant to be run by CI and requires checked out Optimism repo.
+OP_DIR=${OP_DIR:-}
+if [ -z "$OP_DIR" ]; then
+  echo "OP_DIR is not set. Please set it to the contracts-bedrock path of the Optimism repo."
+  exit 1
+fi
+
+# Read environment variables and constants
+source $PWD/scripts/foundry/constants.sh
+
+mkdir -p $ANVIL_FOLDER
+echo "Anvil state will be saved to $ANVIL_FOLDER"
+
+# create package.json
+echo "{\"name\": \"@celo/devchain-anvil\",\"version\": \"1.0.0\",\"repository\": { \"url\": \"https://github.com/celo-org/celo-monorepo\", \"directory\": \"packages/protocol/migrations_sol\" },\"homepage\": \"https://github.com/celo-org/celo-monorepo/blob/master/packages/protocol/migrations_sol/README.md\",\"description\": \"Anvil based devchain that contains core smart contracts of celo\",\"author\":\"Celo\",\"license\": \"LGPL-3.0\"}" > $TMP_FOLDER/package.json
+
+cp $PWD/migrations_sol/README.md $TMP_FOLDER/README.md
```

Comment:
What is `TMP_FOLDER`? We create two files there and it doesn't appear to be used elsewhere in the script?

Reply:
I added comment with explanation, but it is used in `protocol-devchain-anvil.yml`:
```
- name: Prepare package for publishing
        run: yarn utils:prepare-devchain-anvil-publishing
        working-directory: packages/protocol
        env:
          RELEASE_TYPE: ${{ env.RELEASE_TYPE }}
          RELEASE_VERSION: ${{ env.RELEASE_VERSION }}
- name: Upload devchain as artifact
        uses: actions/upload-artifact@v4
        with:
          name: devchain-${{ env.PR_NUMBER }}-${{ steps.date.outputs.date }}
          path: packages/protocol/.tmp
          # Max retention time is 90 days for public repos
          # https://docs.github.com/en/actions/learn-github-actions/usage-limits-billing-and-administration#artifact-and-log-retention-policy
          retention-days: 90
```